modules/claims_api/spec/requests/v2/rswag_veteran_identifier_request_spec.rb

```diff
@@ -55,8 +55,7 @@
           )
 
           before do |example|
-            mock_ccg(scopes) do |auth_header|
-              Authorization = auth_header # rubocop:disable Naming/ConstantName
+            mock_ccg(scopes) do
```

Comment:
I haven't tested it, I fully support this change, is there other places like this we should change too?

Reply:
This was the only place doing this sort of thing with the block argument and it was breaking code by overwriting some module originally assigned to `Authorization` with a hash and then later calling `include Authorization` and throwing an exception "can't include a hash".